packages/portal-proto/src/features/user-flow/workflow/AnalysisWorkspace.tsx

```diff
@@ -15,6 +15,8 @@ import {
   DashboardDownloadContext,
 } from "@/utils/contexts";
 
+import { CountHookRegistry } from "@gff/core";
```

Comment:
```suggestion
import { CountHookRegistry } from "@gff/core";
```

Nitpick: Extra new line

Reply:
Thanks for catching this.